### ISSUE TITLE
Release NPM namespace

### ISSUE THREAD

**Issue by shehi** [browniefed/compass-loader#5]
If this project isn't maintained, and never have had a stable release, could you please release the npm namespace "compass-loader" so that someone else could use it?

**Comment by browniefed** [comment]
I agree, however I would like to release the name to someone who will actually maintain it.

If there is interest from you to re-build this that would be great, however if someone is interested they have yet to contact me.

**Comment by shehi** [comment]
It shouldn't be about THIS package per se. Someone could write totally new package of their own and need the namespace.

However I tend to agree: instead of just releasing the namespace, I'd rather you kept it until someone who has something valuable to serve using it, asks for it.
